examples/gno.land/r/nt/commondao/render.gno

```diff
@@ -1,6 +1,523 @@
 package commondao
 
-func Render(string) string {
-	// TODO: Implement Render support
-	return ""
+import (
+	"std"
+	"strconv"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/mux"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/jeronimoalbi/pager"
+	"gno.land/p/moul/realmpath"
+	"gno.land/p/nt/commondao"
+
+	"gno.land/r/sys/users"
+)
+
+const dateFormat = "Mon, 02 Jan 2006 03:04pm MST"
+
+func Render(path string) string {
+	router := mux.NewRouter()
+	router.HandleFunc("", renderHome)
+	router.HandleFunc("{daoID}", renderDAO)
+	router.HandleFunc("{daoID}/settings", renderSettings)
+	router.HandleFunc("{daoID}/proposals", renderProposalsList)
+	router.HandleFunc("{daoID}/proposals/{proposalID}", renderProposal)
+	router.HandleFunc("{daoID}/proposals/{proposalID}/vote/{address}", renderProposalVote)
+	return router.Render(path)
+}
+
+func renderHome(res *mux.ResponseWriter, _ *mux.Request) {
+	// TODO: Render a header text for users
+	// TODO: Render generic links to vote or execute proposals
+	res.Write("# Common DAO\n")
+}
+
+func renderDAO(res *mux.ResponseWriter, req *mux.Request) {
+	dao := mustGetDAOFromRequest(req)
+
+	// Render header messages
+	if dao.IsDeleted() {
+		res.Write("> ⚠ This DAO has been dismissed\n")
+	}
+
+	// Render header
+	res.Write("# ")
+	res.Write(dao.Name())
+	res.Write("\n")
+
+	if s := dao.Description(); s != "" {
+		res.Write(s)
+		res.Write("\n\n")
+	}
+
+	// Render main menu
+	menu := []string{
+		ufmt.Sprintf("[List Proposals](%s)", daoProposalsURL(dao)),
+		ufmt.Sprintf("[View Settings](%s)", settingsURL(dao)),
+	}
+
+	// TODO: Add submenu with proposal links when proposals support is enabled
```

Comment:
When common proposals are introduced they will have a TX link in the DAO menu.

The common proposals that will be introduced will be the text, DAO members update, SubDAO creation and DAO dismissal proposals.